components/bl-static-breadcrumbs/src/index.js

```diff
@@ -0,0 +1,33 @@
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function StaticBreadcrumbsComponent({ component, eventHandlers, elRef }) {
+  const { classList, display, style, options } = component;
+  const { onItemClick, onMouseOver, onMouseOut } = eventHandlers;
+
+  const onClick = item => {
+    const { pageName, pageData } = item;
+
+    BackendlessUI.Navigator.goToPage(pageName, pageData);
```

Comment:
I'm not sure if that should be default behavior. Also pageData should be formed in handler(he will do it in the moment when item will be clicked, but not in advance).

Reply:
Also, what if onItemClick contain request or other calculation that takes time? I assume goToPage could make it useless